dockerfiles/alpine/broker/Dockerfile

```diff
@@ -16,45 +16,53 @@
 #
 # ------------------------------------------------------------------------
 
-# set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk8:jdk8u192-b12-alpine
-MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
+FROM adoptopenjdk/openjdk8:jdk8u212-b03-alpine
+LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
+      com.wso2.docker.source="https://github.com/wso2/docker-ei/releases/tag/v6.4.0.2"
 
-# set user configurations
+# set Docker image build arguments
+# build arguments for user/group configurations
 ARG USER=wso2carbon
 ARG USER_ID=802
 ARG USER_GROUP=wso2
 ARG USER_GROUP_ID=802
 ARG USER_HOME=/home/${USER}
-# set dependant files directory
-ARG FILES=./files
-# set wso2 product configurations
-ARG WSO2_SERVER=wso2ei
+# build arguments for WSO2 product installation
+ARG WSO2_SERVER_NAME=wso2ei
 ARG WSO2_SERVER_VERSION=6.4.0
-ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}
-ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER_PACK}
+ARG WSO2_SERVER_PROFILE_NAME=broker
+ARG WSO2_SERVER_PROFILE_OPTIMIZER_NUMBER=4
+ARG WSO2_SERVER=${WSO2_SERVER_NAME}-${WSO2_SERVER_VERSION}
+ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}
+ARG WSO2_SERVER_DIST_URL=https://github.com/wso2/product-ei/releases/download/v${WSO2_SERVER_VERSION}/${WSO2_SERVER}.zip
+# build arguments for external artifacts
+ARG DNS_JAVA_VERSION=2.1.8
+ARG K8S_MEMBERSHIP_SCHEME_VERSION=1.0.5
```

Comment:
@Vathsan let's update the version to the latest [`1.0.7`](https://github.com/wso2/docker-is/blob/v5.11.0.1/dockerfiles/ubuntu/is/Dockerfile#L40) version.

Reply:
Done in 532f659